app/controllers/kudos_controller.rb

```diff
@@ -84,6 +101,6 @@ def basic_user_info_by_id(id)
   end
 
   def kudo_params
-    params.require(:kudo).permit(:body, :receiver_email)
+    params.require(:kudo).permit(:body, :receiver_email, :id)
```

Comment:
This makes the kudo id from the request available via kudo_params[:id], following the pattern for accessing kudo attributes from the create method. Is there a better way to do this?